iambic/core/utils.py

```diff
@@ -731,3 +731,57 @@ async def remove_expired_resources(
                 setattr(resource, field_name, new_value)
 
     return resource
+
+
+def recursively_convert_dict_keys(
```

Comment:
I think this function is the same as `normalize_dict_keys`

Reply:
Good call out - It worked like magic.